inductiva/fluids/wind_tunnel/README.md

```diff
@@ -1,28 +1,13 @@
-- `flow_velocity`: Airflow velocity in m/s, e.g. `[30, 0, 0]`. Notice that the airflow is injected from the negative x-direction.
-
-Now, the user is ready to simulate the steady state. Here, the user chooses the object to be inserted inside the wind tunnel. This object is defined with a geometry file in [STL](https://en.wikipedia.org/wiki/STL_(file_format)) or [OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file) format. Notice that:
-- the object is not re-scaled automatically, so the user must either change the dimensions of the domain or scale the object himself.
-- the required meshing step is automatically done on the backend before the simulation starts with [snappyHexMesh tool](https://www.openfoam.com/documentation/user-guide/4-mesh-generation-and-conversion/4.4-mesh-generation-with-the-snappyhexmesh-utility) of OpenFOAM.
-
-To test this scenario we make available one example, but users can use their own files, as long as it is in STL or OBJ format.
-
-The simulation parameters available for the user to configure are:
-- `num_iterations`: Set the maximum number of iterations for the iterative algorithm to converge.
-- `resolution`: Controls the resolution of the meshing that is done prior to the simulation. The higher the resolution, the finer the meshing. Possibilities: "high", "medium", "low".
-
-Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`, `n_cores`.
-Launching a simulation returns a task object, which can be used to verify the status of the simulation, get the simulation outputs and access post-processing tools. See more in the [Tasks section](inductiva/tasks/README.md).
-
+This scenario models the aerodynamics of an object inside a virtual wind tunnel for a given airflow velocity. This scenario models air as an incompressible fluid and thus for the maximum speed available is 100 m/s.
+Air is injected through one side of the wind tunnel and exits through an outlet on the opposite side. The air flow within the tunnel is modified according to the structure of the object. The system is modelled with the steady-state equations for incompressible flow and the $k-\epsilon$ turbulence models. The simulation is currently performed with the OpenFOAM simulator.
 
 ### Example:
 
-To run this example you need an object of OBJ or STL format. See below how to fetch one that we have available in our Github repository.
+**Note:** To access the visualization methods the user needs to install the extra dependencies for the fluids package with `pip install --upgrade "inductiva[fluids_extra]"`.
+
+This example requires an object of OBJ or STL format, we make one available through the code.
 
 Do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
```

Comment:
Maybe instead of putting the link direct the user to the main page where we say how to get the API token.